src/entry-client.js

```diff
@@ -4,34 +4,25 @@ export default function initializeClient(createApp, clientOpts) {
     const opts = Object.assign({
         appSelector: '#app',
         hmr: true,
-        initialTranslationsMetaTag: null,
```

Comment:
This should just be done before calling initializeClient now and assigned to context ins a wrapper around `createApp`